camelot/view/controls/delegates/customdelegate.py

```diff
@@ -223,14 +222,14 @@ def get_standard_item(cls, locale, model_context):
         item.roles[VisibleRole] = model_context.field_attributes.get('visible', True)
         item.roles[NullableRole] = model_context.field_attributes.get('nullable', True)
         item.roles[IsStatusRole] = False
-        # FIXME: move choices to delegates that actually use it?
-        choices = model_context.field_attributes.get('choices')
-        if choices is not None:
-            choices = [CompletionValue(
-                value=initial_naming_context._bind_object(obj),
-                verbose_name=verbose_name
-                )._to_dict() for obj, verbose_name in choices]
-        item.roles[ChoicesRole] = choices
+        # # FIXME: move choices to delegates that actually use it?
```

Comment:
Dit is reeds verplaatst neem ik aan?

Reply:
ik vermoed dat deze code er gekomen is voor de bouw van de table view, met combobox editor 'only', nl, zonder dat de volledige field attributes werden geserialiseerd voor alle delegates